cpp/src/barretenberg/grumpkin_srs_gen/grumpkin_srs_gen.cpp

```diff
@@ -0,0 +1,37 @@
+#include <iostream>
+#include <fstream>
+
+#include "barretenberg/ecc/curves/grumpkin/grumpkin.hpp"
+#include "barretenberg/srs/io.hpp"
+
+/* Generates a monomial basis Grumpkin SRS for testing purposes.
+   We only provide functionality create a single transcript file.
+   The SRS has the form [1]_1, [x]_1, [x^2]_1, ... where x = 2. */
+int main(int argc, char** argv)
```

Comment:
Maybe we should add a TODO to change this logic in the future, since we are not going to use this format in the future but rather sample hashed points?

Reply:
Mentioned in #492